cardinal/system/system_game_setup.go

```diff
@@ -198,6 +227,112 @@ func (s *GameSetup) setupMountainPath(world cardinal.WorldContext) {
 	world.Logger().Debug().Msg("---->Mountain path setup complete")
 }
 
+// setupForge sets up the forge room in the game world.
+func (s *GameSetup) setupForge(world cardinal.WorldContext) {
+	world.Logger().Debug().Msg("---->Setting up the forge.....")
+	// KFORGE -> W
+	open2Barn := s.createAction(enums.ActionTypeOpen, " The broken window is still there"+
+		" just don't cut yourself", true, true, true, 0, 0)
+	plainBarn := []uint32{open2Barn}
+	dObjs := []component.Object{s.createDirObject(enums.DirectionTypeWest, enums.RoomTypeBarn,
+		enums.ObjectTypeWindow, enums.MaterialTypeWood,
+		"window", plainBarn, world)}
+
+	// Petrol Object
+	burn := s.createAction(enums.ActionTypeBurn, " The petrol (with its unique funny smell)"+
+		" burns fiercely with a scent that makes you feel dizzy", true, false, true, 0, 0)
+
+	petrolActions := []uint32{burn}
+	objs := []component.Object{s.createObject(enums.ObjectTypePetrol, enums.MaterialTypeIKEA,
+		"a strange liquid that seems to be petrol, probably there is three liters of it, it's highly burnable though", enums.ObjectTypePetrol.String(), petrolActions, true, world)}
+
+	// Matchsticks Object
+	light := s.createAction(enums.ActionTypeLight, " The matchsticks (despite their small size)"+
+		" lights enough to see a small distance. You have to use them quickly"+
+		" or your fingers will be burn", true, false, true, 0, 0)
+
+	matchesActions := []uint32{light}
+	objs = append(objs, s.createObject(enums.ObjectTypeMatchsticks, enums.MaterialTypeWood,
+		"a group of matchsticks that have survived the pass of time, you can probably light them up, as they seem to be in a good condition", enums.ObjectTypeMatchsticks.String(), matchesActions, true, world))
```

Comment:
"a box of matches that ahve survived the passing of the ages..."

Reply:
done